mypy/checker.py

```diff
@@ -7519,6 +7542,29 @@ def has_valid_attribute(self, typ: Type, name: str) -> bool:
     def get_expression_type(self, node: Expression, type_context: Type | None = None) -> Type:
         return self.expr_checker.accept(node, type_context=type_context)
 
+    def check_deprecated(self, typ: SymbolNode | None, context: Context) -> None:
```

Comment:
The parameter name `typ` is now a bit misleading now.

Reply:
Yes, definitively misleading. Changed it to `node`.